py/src/braintrust/wrappers/_anthropic_utils.py

```diff
@@ -0,0 +1,93 @@
+"""Shared utilities for Anthropic API wrappers."""
```

Comment:
you should start a `claude_agent_sdk` folder (include a `__init__.py`) and that way you can move all the things in there. 

```
./claude_agent_sdk
  ./utils.py
  ...
```